src/plugins/observability/log-pusher.js

```diff
@@ -324,7 +324,7 @@ export class LogPusher {
 
     const metrics1 = [];
     const metrics2 = [];
-    const [version, ip, region, up, qname, qtype, ans, f] = all;
+    const [version, ip, region, qname, qtype, ans, f] = all;
```

Comment:
is `up` not deconstructable by the array `all`?

Reply:
I’m not sure. It seems to be unused and that’s all

Reply:
It may be unused, but it is needed for correctness as destructuring is positional, if that makes sense?